.travis.yml

```diff
@@ -93,14 +93,14 @@ jobs:
     script:
     - export NETWORK=mainnet
     - tar xzf $STACK_WORK_CACHE
-    - stack --no-terminal test cardano-wallet-core:unit
+    - stack --no-terminal test cardano-wallet-http-bridge:unit
```

Comment:
So in CI, we have two jobs running tests. One for testnet which run everything including integration tests. And another one for mainnet, to run a couple of unit tests with a different protocol magic. Yet, we only pattern-match on the protocol magic (and actually, have access to it) in the bridge code so there's no need to re-run the core tests on mainnet.